doc/components/dropdown.md

```diff
@@ -26,6 +26,11 @@ like `text="Click me" icon="icon-filter"`. As following you can see an example o
   </div>
 </div>
 
+**Disabled**
+As any common html element the dropdown can be disabled by adding the `disabled="1"` flag to it.
```

Comment:
Well, as you both discussed this I checked if `disabled="1"` is really valid XHTML specification and it is not. If you use W3C Validator it shows you an error.

> For example, setting the disabled attribute to the value "false" is disallowed, because despite the appearance of meaning that the element is enabled, it in fact means that the element is disabled (what matters for implementations is the presence of the attribute, not its value).

or 

> A number of attributes are boolean attributes. The presence of a boolean attribute on an element represents the true value, and the absence of the attribute represents the false value.

from here:
http://w3c.github.io/html/single-page.html#sec-boolean-attributes 

so basically:

```
<input type="text" disabled />
<input type="text" disabled="" />
<input type="text" disabled="disabled" />
```

See: https://codepen.io/fokusferit/pen/gRoWKd (disabled="{0,1, lala, ...}" everything disables the button.